Packs/DeveloperTools/Integrations/XsoarPowershellTesting/XsoarPowershellTesting.ps1

```diff
@@ -8,11 +8,11 @@ $script:INTEGRATION_ENTRY_CONTEX = "XSOAR"
 
 function TestModuleCommand() {
     # Raw response
-    $raw_response = @{}
+    $raw_response = $null
     # Human readable
-    $human_readable = "Ok!"
+    $human_readable = "ok"
     # Entry context
-    $entry_context = @{}
+    $entry_context = $null
```

Comment:
ma ze

Reply:
test-module always fails (see nightly runs) this is the fix for it.
it failed my build so fixed it here instead of another PR.